package.json

```diff
@@ -4,15 +4,16 @@
   "description": "Experimental native Web Components compiler.",
   "repository": {
     "type": "git",
-    "url": "https://github.com/ProjectEvergreen/wcc.git"
+    "url": "git+https://github.com/ProjectEvergreen/wcc.git",
+    "directory": "packages/wcc"
```

Comment:
we don't have a packages directory though, that only really applies to monorepos.

So just the `url` should be changed here, please remove the addition of `directory`. 👍